CollAction/Views/Projects/Embed.cshtml

```diff
@@ -7,7 +7,49 @@
 @{
     Layout = "_EmbeddedLayout";
     ViewData["Title"] = Localizer["Projecten"];
-    var defaultDisplayModel = FindProjectViewModel.DefaultDisplayProjectViewModel;
 }
 
-<div id="projects-container"></div>
```

Comment:
This one is sufficient for the React component to render the project tile(s). It works the same way as the projects page.